manifests/instance.pp

```diff
@@ -1278,7 +1278,10 @@
       }
 
       # ordering
-      Tomcat::Context["instance_${name}_admin_context"] -> File <| tag == "instance_${name}_catalina_tree" |>
+      File <| tag == "instance_${name}_catalina_tree" |>
```

Comment:
Would something like this work here?

```puppet
File <| tag == ... |>
-> File <| tag == ... |>
```

Reply:
(meant `-> Tomcat::Context <| tag == ... |>` sorry)

Reply:
It could / should, but haven't tested it.

Reply:
I'd either
* order resources using collectors/tags (preferred if supported)
* leave it as is but remove the `instance_${name}_admin_context` tag from `Tomcat::Context` resources, which is now obsoleted by this PR

Apart from that LGTM 👍

Reply:
ok, tested with the collector and works, pr updated